### PR TITLE
Add com.bitstower.Markets

### PR DESCRIPTION
**Pull request by tomasz-oponowicz**
The Markets application delivers financial data to your fingertips. Track stocks, currencies and cryptocurrencies.

### CODE REVIEW

File: com.bitstower.Markets.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+    "app-id" : "com.bitstower.Markets",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.34",
```
Comment (bilelmoussaoui) [inline com.bitstower.Markets.json:4]: Any reason for not using 3.36?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+## Test
```
Comment (bilelmoussaoui) [inline README.md:1]: We tend to avoid duplicating documentation on each application repo. Feel free to remove this file.

**Comment by bilelmoussaoui** [comment]
bot, build com.bitstower.Markets

File: com.bitstower.Markets.json (hunk 1 above)
Comment (tomasz-oponowicz) [reply com.bitstower.Markets.json]: Changed to 3.36.

File: README.md (hunk 2 above)
Comment (tomasz-oponowicz) [reply README.md]: No problem. The file has been removed.

**Comment by bilelmoussaoui** [comment]
bot, build com.bitstower.Markets

**Comment by bilelmoussaoui** [comment]
Anyone else from upstream that would like write access to the repository as well? Otherwise looks good to me

**Comment by tomasz-oponowicz** [comment]
Just me for now.

**Comment by tomasz-oponowicz** [comment]
Thanks!

**Pull request closed without merging**